### DISCUSSION TITLE
请问文章每章写完之后都会有一页空白，请问应该怎么解决？

### DISCUSSION THREAD

**Discussion by dafeixia123456** [note286/xduts#123] [Q&A] [answered]
![36e7e8992515fc334a088e1245b9375](https://user-images.githubusercontent.com/117727115/234762486-4f219948-9ac0-45f0-941c-2a3530aa8e18.png)
![752ce9b1919e172d0b06c4cbe939b59](https://user-images.githubusercontent.com/117727115/234762492-5d19281b-c26e-42e7-952e-114befee1089.png)

**Comment by note286** [comment, marked as answer]
需要双面打印装订。

**Reply by dafeixia123456** [reply]
哦哦好的感谢大佬
